src/report_workflow/test_report_runner.py

```diff
@@ -135,4 +137,17 @@ def generate_test_command(test_type: str, test_manifest_path: str, artifacts_pat
     return command
 
 
+def get_os_cluster_logs(base_path: str, test_number: str, test_type: str, component_name: str, config: str) -> List[str]:
+    os_stdout = ''
+    os_stderr = ''
+    if base_path.startswith("https://"):
+        os_stdout = "/".join([base_path.strip("/"), "test-results", test_number, test_type, component_name, config, "local-cluster-logs/id-0/stdout.txt"])
```

Comment:
I think this might not be `id-0` all the time. CCR team added this `id` for the cases that there are multiple clusters so we may need to iterate through all of them. https://github.com/opensearch-project/opensearch-build/pull/4097
This would also applied to some dashboards integ tests I believe.

Reply:
Just verified with past runs and here is the structure, for all the plugins except CCR the cluster id for `with-security` runs is `id-0` and `without-security` is `id-1`. I have updated the code for the same. 
For CCR, 4 node cluster is spun up, `with-security` has `id-0, id-1, id-2, id-3` and without has 4 - 7. I think it is fine for now wrt CCR as the logs would definitely come handy, even though it comes from one configuration run, in cases where OS process fails to come up.

Reply:
But this structure will also be used for dashboards plugins. From what I'm seeing, `with-security` will have `id-0` and `id-1`; `without-security` will have `id-2` & `id-3`. We might still want to have some approaches to accommodate for dashboards test.

Reply:
Fixed it. @zelinh